src/main/java/org/assertj/core/api/MapAssert.java

```diff
@@ -94,7 +97,19 @@ public MapAssert(Map<KEY, VALUE> actual) {
 
   @SafeVarargs
   @Override
-  public final AbstractListAssert<?, List<? extends Object>, Object, ObjectAssert<Object>> extracting(Function<? super Map<KEY, VALUE>, Object>... extractors) {
+  public final AbstractListAssert<?, List<?>, Object, ObjectAssert<Object>> extracting(Function<? super Map<KEY, VALUE>, Object>... extractors) {
     return super.extracting(extractors);
   }
+
+  @Override
+  public final AbstractListAssert<?, List<?>, Object, ObjectAssert<Object>> extractingFromEntries(Function<? super Map.Entry<KEY, VALUE>, Object> extractor) {
```

Comment:
I think we don't need to override as there is no generic vararg involved with that method.

Reply:
I meant I think we can remove this method from `MapAssert`.
I'm not sure to understand your comment about overriding, if it was not overridden the `@Overidde` would not compile.

Reply:
Oh my bad. I though that you referred another point (which of the last commit on the file). Of course we don't need it to override it.